platform/cavium/cavm-sai.mk

```diff
@@ -1,12 +1,20 @@
 # Cavium SAI
 
+CAVM_SAI_GITHUB = https://github.com/garyachy/OpenXPS/raw/sai_0.9.5/SAI/cavm-sai/
+
 CAVM_LIBSAI = libsai.deb
-$(CAVM_LIBSAI)_PATH = $(PLATFORM_PATH)/cavm_sdk
 CAVM_SAI = sai.deb
-$(CAVM_SAI)_PATH = $(PLATFORM_PATH)/cavm_sdk
 XP_TOOLS = xp-tools.deb
-$(XP_TOOLS)_PATH = $(PLATFORM_PATH)/cavm_sdk
 XPSHELL = xpshell.deb
-$(XPSHELL)_PATH = $(PLATFORM_PATH)/cavm_sdk
 
-SONIC_COPY_DEBS += $(CAVM_LIBSAI) $(CAVM_SAI) $(XP_TOOLS) $(XPSHELL)
+$(CAVM_LIBSAI)_URL = $(CAVM_SAI_GITHUB)/$(CAVM_LIBSAI)
+$(CAVM_SAI)_URL = $(CAVM_SAI_GITHUB)/$(CAVM_SAI)
+$(XP_TOOLS)_URL = $(CAVM_SAI_GITHUB)/$(XP_TOOLS)
+$(XPSHELL)_URL = $(CAVM_SAI_GITHUB)/$(XPSHELL)
+SONIC_ONLINE_DEBS += $(CAVM_LIBSAI) $(CAVM_SAI) $(XP_TOOLS) $(XPSHELL)
+
+#$(CAVM_LIBSAI)_PATH = $(PLATFORM_PATH)/cavm_sdk
```

Comment:
You can just copy files to target/debs, and make won't try to download them even if they are in SONIC_ONLINE_DEBS group.
Using SONIC_COPY_DEBS is an unnecessary step here.